### PR TITLE
Support GHC 8.10

### PR DESCRIPTION
**Pull request by junjihashimoto**
We can compile hasktorch except for codegen. 
codegen depends on blaze-markup which does not support ghc8.10.
https://github.com/jaspervdj/blaze-markup/pull/49

### CODE REVIEW

File: examples/examples.cabal [hunk 1]
```diff
@@ -141,13 +141,13 @@ executable vae
...
-  ghc-options:         -fno-warn-partial-type-signatures
-  build-depends:       bytestring
-                     , serialise
+-- executable serialization
+--   import:              config
+--   hs-source-dirs:      serialization
+--   main-is:             Main.hs
+--   ghc-options:         -fno-warn-partial-type-signatures
+--   build-depends:       bytestring
+--                      , serialise
```
Comment (junjihashimoto) [inline examples/examples.cabal:150]:
```
$ cabal v2-build examples
Resolving dependencies...
cabal: Could not resolve dependencies:
[__0] trying: blaze-markup-0.8.2.4 (user goal)
[__1] trying: base-4.14.0.0/installed-4.1... (dependency of blaze-markup)
[__2] trying: examples-0.2.0.0 (user goal)
[__3] next goal: serialise (dependency of examples)
[__3] rejecting: serialise-0.2.2.0, serialise-0.2.1.0, serialise-0.2.0.0
(conflict: base => ghc-prim==0.6.1/installed-0.6..., serialise =>
ghc-prim>=0.3.1.0 && <0.6)
[__3] rejecting: serialise-0.1.0.0 (conflict: base =>
ghc-prim==0.6.1/installed-0.6..., serialise => ghc-prim>=0.3 && <0.6)
[__3] fail (backjumping, conflict set: base, examples, serialise)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, blaze-markup, serialise,
examples
Try running with --minimize-conflict-set to improve the error message.
```

**Comment by austinvhuang** [comment]
LGTM thank you @junjihashimoto

**Comment by junjihashimoto** [comment]
Thx!

**Comment by tycho01** [comment]
oh, looks like there was still a GHC version reference in Nix as well, see #362.

**Pull request merged**